main.py

```diff
@@ -0,0 +1,83 @@
+from telegram import KeyboardButton, ReplyKeyboardMarkup
+from telegram.ext.updater import Updater
+from telegram.update import Update
+from telegram.ext.callbackcontext import CallbackContext
+from telegram.ext.commandhandler import CommandHandler
+from telegram.ext.messagehandler import MessageHandler
+from telegram.ext.filters import Filters
+from services import weather_service
+import sqlite3
+from datetime import datetime, timedelta
+import atexit
+
+updater = Updater("5440718277:AAFPwMvNactHwIkrZa00SAOicH2ddCbRyNs",
+                  use_context=True)
+
+con = None
+cur = None
+
+
+def close_connection():
+    cur.close()
+    con.close()
+
+
+def start(update: Update, context: CallbackContext):
+    update.message.reply_text("Hello welcome to the weather bot", reply_markup=main_menu_keyboard())
+
+
+def get_user_data(username):
+    res = cur.execute(f"SELECT * FROM Location WHERE username='{username}'")
```

Comment:
that's SQL injection right there, here's good article on it https://realpython.com/prevent-python-sql-injection/